### ISSUE TITLE
Support for ALT/CMD + Left arrow key to go back to previous page 

### ISSUE THREAD

**Issue by jalilmotaz** [ytmdesktop/ytmdesktop#365]
- [ ] I understand that **YTMDesktop have NO affiliation with Google or YouTube**.
- [ ] I verified that there is no open issue for the same subject.

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by jippylong12** [comment]
This should now be addressed in #1709, which adds configurable Navigate Back / Navigate Forward global shortcuts under Settings > Shortcuts. Users can bind them to whatever keys they prefer (e.g. Cmd+Left/Right on Mac, Alt+Left/Right on Windows/Linux).

Note: #1144 was previously closed as a duplicate of this issue.

This can likely be closed once the PR is merged.
